Marlin/stepper.cpp

```diff
@@ -105,16 +105,16 @@ static volatile bool endstop_z_probe_hit = false; // Leaving this in even if Z_P
   static bool old_z_max_endstop = false;
 #endif
 #ifdef Z_DUAL_ENDSTOPS
-  #if HAS_Z2_MIN
+  // #if HAS_Z2_MIN
```

Comment:
Does `Z_DUAL_ENDSTOPS` require at least one of these Z2 settings to exist, but not both? Then, perhaps the missing one can be defined as a macro, instead of just always including both here.